src/containers/EditPalette/EditPalette.test.js

```diff
@@ -50,16 +70,23 @@ describe('EditPalette', () => {
     });
   });
 
-  describe('mapDispatchToProps', () => {
-    it('calls dispatch with a setPaletts action', () => {
-      const mockDispatch = jest.fn()
-      const mockPalettes = ['palettes']
-      const actionToDispatch = setPalettes(mockPalettes)
+  describe.skip('mapDispatchToProps', () => {
```

Comment:
Are these skipped because they aren't passing? I'll pull the code and take a look. I think we can structure them slightly different to have them pass.